src/ResourceManager/ApiManagement/Commands.ApiManagement.ServiceManagement/Commands.ApiManagement.ServiceManagement.csproj

```diff
@@ -24,7 +24,7 @@
     <WarningLevel>4</WarningLevel>
     <RunCodeAnalysis>true</RunCodeAnalysis>
     <UseVSHostingProcess>true</UseVSHostingProcess>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
@solankisamir why were these changed to `false`?

Reply:
Marking a parameter/cmdlet as Obsolete was causing Build Errors, as they come as Warning.

Reply:
Marking a parameter/cmdlet as Obsolete was causing Build Errors, as they come as Warning.

---
In reply to: [188763417](https://github.com/Azure/azure-powershell/pull/6189#discussion_r188763417) [](ancestors = 188763417)